### ISSUE TITLE
Updated to new comfy portable and its broken again despite all dependencies installed

### ISSUE THREAD

**Issue by 1blackbar** [807502278/ComfyUI_MaskGCT#21]
There is something that  breaks this too easily... its stuck after it loads models

Loading openai/whisper-large-v3-turbo...
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
You have passed task=transcribe, but also have set `forced_decoder_ids` to [[1, None], [2, 50360]] which creates a conflict. `forced_decoder_ids` will be ignored in favor of task=transcribe.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token.As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Loading config.json...
Loading model.safetensors...
Loading preprocessor_config.json...
Loading semantic_codec/model.safetensors...
Loading acoustic_codec/model.safetensors...
Loading acoustic_codec/model_1.safetensors...
Loading t2s_model/model.safetensors...
Loading s2a_model/s2a_model_1layer/model.safetensors...
Loading s2a_model/s2a_model_full/model.safetensors...
  0%|                                                                                            | 0/1 [00:00<?, ?it/s]words count mismatch on 100.0% of the lines (1/1)

**Comment by 1blackbar** [comment]
This version works fine so whatever you did after that - broke it 

 https://github.com/807502278/ComfyUI_MaskGCT/tree/3a5961f4e88f37fe86b9eb5d1f0a2e5540eeef14

**Comment by 807502278** [comment]
Thank you for the feedback. I apologize for the inconvenience, but I've recently had some changes in my work and I'm currently unable to maintain it. Once I settle down, or if you can make the changes, feel free to submit a PR.
